packages/now-cli/src/util/dev/server.ts

```diff
@@ -533,7 +533,7 @@ export default class DevServer {
     // The default empty `vercel.json` is used to serve all files as static
     // when no `vercel.json` is present
     let configPath = 'vercel.json';
-    let config: NowConfig = this.cachedNowConfig || {
+    let config: NowConfig = (canUseCache && this.cachedNowConfig) || {
```

Comment:
I'm confused, wouldn't this line never evaluate to true `(canUseCache && this.cachedNowConfig)` because of line 527-529? It seems like it would always create a new object so you could remove the conditional here and always assign the new object.

Reply:
The case is that `canUseCache` is set to false, so that first `if` branch isn't true, and then we want to _not_ use `cachedNowConfig` when it's set to false here on this line.

Reply:
But if the first `if` branch is false, wouldn't this second branch (line 536) also be false?

Or is there something that can happen in between asynchronously to assign the cached now config?

Reply:
`cachedNowConfig` may exist, but `canUseCache` is set to false.

Reply:
I'm suggesting doing this instead:

```js
let config = {
  version: 2,
  [fileNameSymbol]: configPath,
};
```